crates/sui-config/Cargo.toml

```diff
@@ -17,17 +17,17 @@ serde_yaml = "0.8.26"
 rand = "0.8.5"
 dirs = "4.0.0"
 multiaddr = "0.14.0"
-once_cell = "1.13.1"
+once_cell = "1.14.0"
 tracing = "0.1.36"
 
-fastcrypto = { git = "https://github.com/MystenLabs/fastcrypto", rev = "c022a2ae23ca7cc2778293fd3b1db42e8cd02d3b" }
+fastcrypto = "0.1"
```

Comment:
In NW we specifically pin the minor version as well 0.1.2. Shall we do the same here?

Reply:
Scoping down this PR.